src/Language/Haskell/LSP/Test/Decoding.hs

```diff
@@ -120,6 +121,7 @@ matchResponseMsgType req = case req of
   where decoded x = fromMaybe (error $ "Couldn't decode response for the request type: "
                                         ++ show req ++ "\n" ++ show x)
                               (decode x)
+        removeNullResult x = maybe x (<> "}") (B.stripSuffix ",\"result\"=null}" x)
```

Comment:
An ugly hack to sanitize the response from the manual javascript session
Maybe we should have a generic way to remove fields with null values...